eng/Versions.props

```diff
@@ -32,9 +32,10 @@
     <CodeStyleLayerCodeAnalysisVersion>3.7.0</CodeStyleLayerCodeAnalysisVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20407.3</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.8.0-2.20414.4</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.4.248</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.8.39</VisualStudioEditorPackagesVersion>
+    <VisualStudioShellPackagesVersion>16.7.30021.50</VisualStudioShellPackagesVersion>
     <ILToolsPackageVersion>5.0.0-alpha1.19409.1</ILToolsPackageVersion>
-    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.8.5</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
```

Comment:
📝 The TypeScript package ships version 16.8.52, but advertises it as 16.8.5 in **TypeScriptVisualStudio.pkgdef**. This authoring error renders 16.8.5 unusable within Visual Studio.

Reply:
It looks like this problem is unique to 16.8 Preview 3, so for now I've removed this change from the pull request.